controls/roles/manage-service/molecule/ssv-prysm/prepare.yml

```diff
@@ -57,17 +57,6 @@
         mode: 0644
       become: yes
 
-    - block:
```

Comment:
without fastsync? I thought beacon chain should be fully synced

Reply:
I don't think this is necessary because we can check also without a fully synced beacon if the bloxssv connection works